docs/examples/parameters/path_parameters_2.py

```diff
@@ -12,7 +12,7 @@ class Order(BaseModel):
 
 
 ORDERS_BY_DATETIME = {
-    datetime.fromtimestamp(1667924386, tz=timezone.utc): [
+    datetime.fromtimestamp(1667924386): [
```

Comment:
Why do we do `datetime.fromtimestamp(1667924386)` here? I am not understanding the epoch significance of 2022-09-08 21:13:06

Reply:
Prob just when the test was first written.